crypto/ec/ec_curve.c

```diff
@@ -3090,9 +3107,10 @@ static EC_GROUP *ec_group_new_from_data(const ec_list_element curve)
 
     /* If no curve data curve method must handle everything */
     if (curve.data == NULL)
-        return EC_GROUP_new(curve.meth != NULL ? curve.meth() : NULL);
+        return EC_GROUP_new_ex(libctx,
+                               curve.meth != NULL ? curve.meth() : NULL);
 
-    if ((ctx = BN_CTX_new()) == NULL) {
+    if ((ctx = BN_CTX_new_ex(libctx)) == NULL) {
```

Comment:
Is `ERR_R_MALLOC_FAILURE` in the line below still relevant with the new `BN_CTX_new_ex()`?

Reply:
It's still the only reason that `BN_CTX_new_ex()` will fail, so I think it is ok.